apps/bare-expo/ios/BareExpo/AppDelegate.swift

```diff
@@ -7,17 +7,15 @@
 //
 
 import Foundation
+import EXDevMenu
 
 @UIApplicationMain
-class AppDelegate: UMAppDelegateWrapper {
+class AppDelegate: UMAppDelegateWrapper, DevMenuDelegateProtocol {
```

Comment:
Looking good. It seems like there is no Objective-C version for this protocol yet? 🤔 Was going to add it to dev client demo then ran into that.

Reply:
This is usable from Objective-C actually, nvm!